validation/src/main/java/io/micronaut/validation/validator/DefaultConstraintDescriptor.java

```diff
@@ -0,0 +1,123 @@
+/*
```

Comment:
This class was internal for IntrospectedBeanDescriptor,  just exposed, no real code change here.